EDMarketConnector.py

```diff
@@ -1489,6 +1565,11 @@ def crewroletext(role: str) -> str:
                 if not should_return:
                     self.w.after(int(SERVER_RETRY * 1000), self.capi_request_data)
 
+            if entry['event'] in ['CarrierBuy', 'CarrierStats', 'CarrierTradeOrder']:  # 'CargoTransfer' too?
```

Comment:
```suggestion
            if entry['event'] in ('CarrierBuy', 'CarrierStats', 'CarrierTradeOrder'):  # 'CargoTransfer' too?
```

While its not likely to be changed / changable, good practice to use tuples here

Reply:
Re: using `_` ... to be fair, that wasn't caught when *I* recently added the CAPI killswitches.  It'll need changing in general for those.

Reply:
Want me to do that @Athanasius ?

Reply:
Yes, it's the more correct form.  In general if A_D requests something and I haven't objected, take it as canon.  They know more Python than I do ;) .

Reply:
Ok, tried this and it immediately throws an exception:

```
self.ship_label['text'] = _('Ship') + ':'  # LANG: 'Ship' label in main UI
                              ^
UnboundLocalError: cannot access local variable '_' where it is not associated with a value
```

This exception isn't thrown anywhere near any of the instances of 
```python
should_return, new_data = killswitch.check_killswitch('capi.request.fleetcarrier', {})
```
that I changed to:
```python
should_return, _= killswitch.check_killswitch('capi.request.fleetcarrier', {})
```

So, I suspect this is because `_()` is defined by i18n [as described here](https://stackoverflow.com/questions/5893163/what-is-the-purpose-of-the-single-underscore-variable-in-python/5893946#5893946)

That article suggests using double underscore `__` as an alternative for throwaway variables when used in conjunction with l18n.  @A-UNDERSCORE-D would that be acceptable?

Reply:
Yup, if `__` works, then we'll have to use that (and I'm surprised where I changed it apparently hasn't blown up yet in this branch, but I've only just gotten to test-running it).

Reply:
Ok, will stick with that.